src/make_certs.erl

```diff
@@ -112,8 +112,14 @@ intermediateCA(Root, CA, ParentCA) ->
 endusers(Root, CA, Users) ->
     [enduser(Root, CA, User) || User <- Users].
 
-enduser(Root, CA, User) -> 
-    UsrRoot = filename:join([Root, User]),
+enduser(Root, CA, {User, Path}) ->
```

Comment:
Minor, but I'd prefer if the `Path` parameter was named something like `DirName`, as it's only one segment of the filesystem path.